packages/collaborator-client/src/client.ts

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import core, { Class, CollaborativeDocumentId, Doc, Ref, WorkspaceId, concatLink } from '@hcengineering/core'
+import { MinioService } from '@hcengineering/minio'
+import { LiveQuery } from '@hcengineering/query'
+import { ServerKit, htmlToYDoc } from '@hcengineering/text'
+import { Extensions } from '@tiptap/core'
+import { Doc as YDoc, applyUpdate, encodeStateAsUpdate } from 'yjs'
+
+/**
+ * @public
+ */
+export interface CollaboratorClient {
+  update: (classId: Ref<Class<Doc>>, docId: Ref<Doc>, attribute: string, value: string) => Promise<void>
+  close: () => Promise<void>
+}
+
+/**
+ * @public
+ */
+export function getClient (
+  lq: LiveQuery,
```

Comment:
I supposed it will be more high level, like reuse our current connection to server and use connection to collaborator, how it could be used from UI for issue creation then we will switch to fully minio description's?

Reply:
removed minio and livequery usages